AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+`romcal` was orginally developed by **Julian Pereira** from Malaysia in July 2014.
+
+Since then, many significant contributions have been made towards the progress of this library.
+
+This list recognizes contributors to `romcal` past and present.
+
+#### Active Contributors
+
+as of February 2020
+
+- **Etienne Magnier**
```

Comment:
I would create links to their respective GitHub profiles.

Reply:
Done!